src/test/java/org/jenkinsci/plugins/pipeline/utility/steps/fs/TeeStepTest.java

```diff
@@ -76,6 +76,47 @@ public void smokes() throws Exception {
         });
     }
 
+    @Test
```

Comment:
If this is verifying a fix for a specific bug, it should be marked with `@Issue`. Though if I understand correctly, this is just asserting that new `tee` steps wipe out the previous file contents, which is not a fix for a reported issue. (Whether it is even _desirable_, I am unsure. I would be inclined to leave the original behavior, since you can always delete the file before running `tee` if that is what you wanted.)